oe_theme.theme

```diff
@@ -1504,6 +1504,12 @@ function oe_theme_theme_suggestions_page_alter(array &$suggestions, array $varia
   }
 }
 
+function oe_theme_theme_suggestions_pattern_field_list_alter(array &$suggestions, array $variables): void {
```

Comment:
I've found hook_ui_patterns_suggestions_alter https://ui-patterns.readthedocs.io/en/8.x-1.x/content/developer-documentation.html

Reply:
I change this to use hook_ui_patterns_suggestions_alter